### DISCUSSION TITLE
Library for eigenvalues

### DISCUSSION THREAD

**Discussion by greazee** [OpenRadioss/OpenRadioss#3135] [Q&A]
Hello, i found out that the eigenvalues calculation (/EIG) doesn't work due to lack of some library which is named like DNC, could you share what this kind of library, cause the subroutine IMP_EIGSOL not defined anywhere else in the source code.
![image](https://github.com/user-attachments/assets/0aaf2423-30a3-449c-b85d-d3c75e187107)

**Comment by laurent-altr** [comment]
Hello @greazee, 
In general, code sections disabled by #ifdef are not shared for a variety of reasons. 
These include:
- Dependence on a third-party library that is incompatible with our open-source license.
- Incompatibility with free compilers, such as GFortran.
- Deprecated features or functionality that is not working as intended.
- ...
